### PR TITLE
Add coinlore connector

### PR DESCRIPTION
**Pull request by gendzucuuu**

### CODE REVIEW

File: src/main/java/by/gendzucuuu/cryptocurrencywatcher/CryptoCurrencyWatcherApplication.java [hunk 1]
```diff
@@ -1,8 +1,11 @@
 package by.gendzucuuu.cryptocurrencywatcher;
 
+import by.gendzucuuu.cryptocurrencywatcher.model.News;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
+import java.util.List;
+
```
Comment (Caromeen) [inline src/main/java/by/gendzucuuu/cryptocurrencywatcher/CryptoCurrencyWatcherApplication.java]: ?

File: src/main/java/by/gendzucuuu/cryptocurrencywatcher/connector/impl/CoinloreConnectorImpl.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
+package by.gendzucuuu.cryptocurrencywatcher.connector.impl;
+
+import by.gendzucuuu.cryptocurrencywatcher.connector.CoinloreConnector;
+import by.gendzucuuu.cryptocurrencywatcher.connector.dto.CoinloreInfoResponseDto;
+import by.gendzucuuu.cryptocurrencywatcher.model.Cryptocoin;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.SneakyThrows;
```
Comment (Caromeen) [inline src/main/java/by/gendzucuuu/cryptocurrencywatcher/connector/impl/CoinloreConnectorImpl.java:8]: ?

File: src/main/java/by/gendzucuuu/cryptocurrencywatcher/model/News.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class News {
```
Comment (Caromeen) [inline src/main/java/by/gendzucuuu/cryptocurrencywatcher/model/News.java]: ?

File: src/test/resources/coinlore/connector/findall/btcresponce.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
+[
+  {
```
Comment (Caromeen) [inline src/test/resources/coinlore/connector/findall/btcresponce.json]: filenameisnotreadableusesomeseparotorsforexamplefind-allotunderscorefind_allandbtc-responce

**Review by Caromeen — CHANGES_REQUESTED** [review]

File: src/main/java/by/gendzucuuu/cryptocurrencywatcher/CryptoCurrencyWatcherApplication.java (hunk 1 above)
Comment (gendzucuuu) [reply src/main/java/by/gendzucuuu/cryptocurrencywatcher/CryptoCurrencyWatcherApplication.java]: fixed

File: src/main/java/by/gendzucuuu/cryptocurrencywatcher/connector/impl/CoinloreConnectorImpl.java (hunk 2 above)
Comment (gendzucuuu) [reply src/main/java/by/gendzucuuu/cryptocurrencywatcher/connector/impl/CoinloreConnectorImpl.java:8]: fixed

File: src/main/java/by/gendzucuuu/cryptocurrencywatcher/model/News.java (hunk 3 above)
Comment (gendzucuuu) [reply src/main/java/by/gendzucuuu/cryptocurrencywatcher/model/News.java]: deleted

File: src/test/resources/coinlore/connector/findall/btcresponce.json (hunk 4 above)
Comment (gendzucuuu) [reply src/test/resources/coinlore/connector/findall/btcresponce.json]: fixed

**Review by Caromeen — APPROVED** [review]
Good

**Pull request merged**